bims/static/js/views/location_site_detail.js

```diff
@@ -109,6 +109,162 @@ define(['backbone', 'ol', 'shared'], function (Backbone, ol, Shared) {
                     Shared.Dispatcher.trigger('sidePanel:updateSidePanelHtml', {});
                 }
             });
+            if(isHealthyrivers) {
```

Comment:
move this to new function, and no need to do condition in here, because we use healthyrivers-map.html